<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>packetWidget</class>
 <widget class="QWidget" name="packetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transfer Packet via SystemIF</string>
  </property>
  <widget class="QLabel" name="dataLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Data (hex):</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Address (hex):</string>
   </property>
  </widget>
  <widget class="QLabel" name="routeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Route Info:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects the local address of the flit recipient...</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataEdit4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>data[3]: 64 bit data word to write / Read Result</string>
   </property>
   <property name="text">
    <string>0000000000000000</string>
   </property>
  </widget>
  <widget class="QComboBox" name="subDestIDBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects the destination Sub-ID...</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>CRB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MPB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CORE0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CORE1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PERIN</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QComboBox" name="CMDPrefixBox">
      <property name="whatsThis">
       <string>Selects the command to transfer when the &quot;Transfer&quot; button gets hit...</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>NOGEN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BLOCK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BCMC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BCTILE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BCBOARD</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="CMDBox">
      <property name="whatsThis">
       <string>Selects the command to transfer when the &quot;Transfer&quot; button gets hit...</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>DATACMP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NCDATACMP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MEMCMP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ABORT_MESH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RDO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WCWR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WBI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NCRD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NCWR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NCIORD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NCIOWR</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="txButton">
      <property name="whatsThis">
       <string>Transfer to selected Destination...</string>
      </property>
      <property name="text">
       <string>Transfer now</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="doneButton">
      <property name="whatsThis">
       <string>Quit this dialog...</string>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="addressBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects predefined CRB addresses...</string>
   </property>
   <item>
    <property name="text">
     <string>CUSTOM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="routeBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects the destination tile...</string>
   </property>
   <item>
    <property name="text">
     <string>Tile x=0, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=0, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=0, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=0, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=1, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=2, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=3, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=4, y=3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tile x=5, y=3</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="radioButtonHex">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>78</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex Data</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonBin">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>97</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bin Data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataEdit3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>data[2]: 64 bit data word to write / Read Result</string>
   </property>
   <property name="text">
    <string>0000000000000000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataEdit2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>data[1]: 64 bit data word to write / Read Result</string>
   </property>
   <property name="text">
    <string>0000000000000000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataEdit1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>data[0]: 64 bit data word to write / Read Result</string>
   </property>
   <property name="text">
    <string>0000000000000000</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SIFSubIDBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Selects the System Interface Destination ID...</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="SIFSubIDLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>System IF SubID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="byteEnLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Byte Enable (hex):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="byteEnEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Byte Enable...</string>
   </property>
   <property name="text">
    <string>ff</string>
   </property>
  </widget>
  <widget class="QLabel" name="CMDLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Command:</string>
   </property>
  </widget>
  <widget class="QLabel" name="subDestIDLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination SubID:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>routeBox</tabstop>
  <tabstop>subDestIDBox</tabstop>
  <tabstop>SIFSubIDBox</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>addressBox</tabstop>
  <tabstop>byteEnEdit</tabstop>
  <tabstop>dataEdit4</tabstop>
  <tabstop>dataEdit3</tabstop>
  <tabstop>dataEdit2</tabstop>
  <tabstop>dataEdit1</tabstop>
  <tabstop>txButton</tabstop>
  <tabstop>doneButton</tabstop>
  <tabstop>radioButtonHex</tabstop>
  <tabstop>radioButtonBin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>doneButton</sender>
   <signal>clicked()</signal>
   <receiver>packetWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
